templates/registration/login.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form method="post" action="{% url 'login' %}">
```

Comment:
The form action URL uses the name 'login'. Ensure that this URL name is correctly defined in your URL configuration. If using Django's built-in authentication views, the URL name might be 'login' or 'auth_login', depending on your setup.

Reply:
Everything works for me. According to the assignment.